templates/superwerker.template.yaml

```diff
@@ -151,19 +141,10 @@ Conditions:
   UsingDefaultBucket: !Equals [!Ref QSS3BucketName, 'aws-quickstart']
   IncludeNotifications: !Not [!Equals [ !Ref NotificationsMail, '' ]]
   IncludeBudget: !Equals [ !Ref IncludeBudget, 'Yes' ]
-  IncludeControlTower: !Equals [ !Ref IncludeControlTower, 'Yes' ]
-  IncludeGuardDuty: !And
```

Comment:
@Reb-B it might make sense to split out the reformatting into another commit / PR, so that this commit/PR is scoped to "remove feature flag since Control Tower is a mandatory feature". What do you think?

Reply:
Forget my comment, I was blind :)